addons/website_hr_recruitment/models/hr_job.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Job(models.Model):
+    _inherit = "hr.job"
+
+    job_details = fields.Html('Process Details',
```

Comment:
This column should be created in an upgrade script IMO, otherwise it will set the default value on existing dbs and it will probably not match the customer's processes